### ISSUE TITLE
Automatisation des réponses

### ISSUE THREAD

**Issue by Netinq** [C0casio45/bot_onet#23]
## Contexte
Constatant que souvent les joueurs ont les même question : demande de unban, report... il faudrait permettre une automatisation de certaines réponses. D'une part cela éviterait un travail redondant de l'équipe de support, et d'autre part ça permettrait aux joueurs d'avoir une réponse plus rapide.

### Les cas de figures traités par la fonctionnalité sont les suivants : 

- En tant que joueur, je veux être unban, le bot réponds avec le lien du formulaire, afin d'améliorer la rapidité de réponse et de permettre aux supports de ne plus traiter ces sujets.
- En tant que joueur, je veux report quelqu'un, le bot analyse si le report contiens bien toutes les informations demandées. Le cas contraire il demande au joueur de les renseigner afin d'améliorer le travail du support pour qu'ils n'aient pas à effectuer le travail redondant de demander ces informations.

**Comment by C0casio45** [comment]
Très bonne idée, on passerai par les messages en les analysant et en y cherchant des mots clés ? A voir si le mec demande a ce que l'on ban quelqu'un != le mec s'est fait ban

**Comment by Netinq** [comment]
Je pensai à analyser les messages avec des REGEX.
> A voir si le mec demande a ce que l'on ban quelqu'un != le mec s'est fait ban

Tu voudrai qu'on check si les gens signalés dans le ticket sont déjà sanctionnés ?

**Comment by C0casio45** [comment]
Non, je voulais dire par la que analyser que le message peu preter a confusion
Le mieux serai qu'a l'arrivé, le bot demande ce qui est voulu : unban ou autre et le bot réagis en conséquence, il demande les différentes choses une a une ou d'un coup et si i y a des choses qui manquent il les redemande

**Comment by Netinq** [comment]
Je te propose de faire un mix des deux, je m'explique.

Je bot analyse le message et crois reconnaître une demande de unban. Il va alors poser la question à l'utilisateur : "voulez vous effectuer une demande de débanissement" et l'utilisateur pourras cliquer sur OUI ou NON.

Par rapport aux reports en revanche, je doute que la bonne solution soit de faire un formulaire pour pour le /ban. A mon avis cela détériorerait l'expérience utilisateur. Je pense qu'il faudrait plutôt tenter la REGEX. Ou sinon nous pouvons utiliser la technologie LDA [https://www.npmjs.com/package/lda](https://www.npmjs.com/package/lda )

**Comment by C0casio45** [comment]
A voir, tente le coup je suis chaud
on fera des tests
